admin/tools/docker/4_build-configured-images.sh

```diff
@@ -46,10 +48,11 @@ EOD
 
 # Get the options
 DOCKER_IMAGE="$DEFAULT_DOCKER_IMAGE"
```

Comment:
It seems like this could be simplified to remove `DEFAULT_DOCKER_IMAGE` by initializing `DOCKER_IMAGE` with the default value.